src/difference.js

```diff
@@ -4,7 +4,7 @@ var _curry2 = require('./internal/_curry2');
 
 /**
  * Finds the set (i.e. no duplicates) of all elements in the first list not
- * contained in the second list.
+ * contained in the second list (in terms of R.contains).
```

Comment:
Rather than revealing the implementation then, I would recommend describing the semantics. e.g. "Elements are compared in terms of *value* equality, not *reference* equality."